ProcessMaker/processmaker
# Implement Screen Template Selection and Application in Create New Screen

This PR introduces the functionality to select a screen template and apply it to a new screen during the creation process. It includes UI enhancements for hover and active effects on the screen template cards, along with backend logic to apply template settings to the newly created screen.

## Changes Made:

- Implemented UI updates to enhance user experience when hovering over and selecting screen templates.
- Added a new method applyTemplateSettings to the ScreenController store method to apply template settings to the newly created screen.
- Integrated backend logic to correctly apply template settings based on the selected template.

## How to Test

1. Ensure there are existing screen templates with inputs and custom CSS.
2. Navigate to Designer > Screen and click on '+ Screen' to create a new screen.
3. Hover over different screen templates and verify that hover effects are applied.
4. Select a template and ensure that it becomes active, indicating it's selected.
5. Fill out the form with necessary details.
6. Click on the 'Save' button to create the screen.
7. Verify that the screen is created using the selected template with all associated settings and configurations applied correctly.

## Related Tickets & Packages
- [FOUR-13495](https://processmaker.atlassian.net/browse/FOUR-13495)

ci:next
ci:deploy

## Code Review Checklist


[FOUR-13495]: https://processmaker.atlassian.net/browse/FOUR-13495?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
